src/main/java/com/bulkbuy/enterprise/BulkBuyController.java

```diff
@@ -15,14 +19,35 @@ public String index() {
         return "start";
     }
 
+    /**
+     * Endpoint handler for /newOrder
+     * @return the page newOrder.html
+     */
     @RequestMapping("/newOrder")
-    public String newOrder(){
+    public String newOrder(Model model){
+        Order order = new Order();
+        model.addAttribute(order);
         return "newOrder";
     }
 
+    /**
+     * Endpoint handler for /orderLookup
+     * @return the page orderLookup.html
+     */
```

Comment:
Added Javadoc for the endpoint handler.
Source: https://www.tutorialspoint.com/java/java_documentation.htm